srcs/request_handling/RequestHandler.cpp

```
@@ -1,20 +1,13 @@
 #include "RequestHandler.hpp"
 
-std::unique_ptr<ResponsePacket> handle_request(const std::string &request, const std::shared_ptr<std::vector<ServerConfig>> &available_configs)
+std::unique_ptr<ResponsePacket> handle_request(const RequestPacket &request, const std::shared_ptr<std::vector<ServerConfig>> &available_configs)
```

cppcheck

[style] The function 'handle_request' is never used.

(unusedFunction)